### PR TITLE
DOCS-1422: Change robot to machine pt 2

### PR DESCRIPTION
**Pull request by JessamyT**
Combed through each remaining instance pretty well; sometimes VSC bugs and puts "machinene" or some such instead; tried to catch all those. Left a lot of tutorial usages as-is.

Remaining to-dos (perhaps in separate PR):
- Need to check on CLI and other uses of "robot API key" and similar.
- Comb screenshots and diagrams

### CODE REVIEW

File: docs/build/program/apis/robot.md [hunk 1]
```diff
@@ -248,8 +248,8 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 ### WithChannel
 
-Create a robot that is connected to a machine over the given channel.
-Any robots created using this method will NOT automatically close the channel upon exit.
+Create a machine that is connected to a machine over the given channel.
```
Comment (sguequierre) [inline docs/build/program/apis/robot.md:251]: If we refer to it as robot client in AtAddress we should also use that phrasing here, both methods create RobotClients. Fine w/ referring to it as robot client or machine

File: static/include/services/apis/robot.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
 | Method Name                                                     | Description                                                                  |
 | --------------------------------------------------------------- | ---------------------------------------------------------------------------- |
 | [`Options.with_api_key`](/build/program/apis/robot/#optionswith_api_key) | Create Robot client connection options with an API key as credentials. |
-| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a robot client that is connected to the robot at the provided address. |
+| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a robot client that is connected to the machine at the provided address. |
 | [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a robot that is connected to a robot over the given channel. |
```
Comment (sguequierre) [inline static/include/services/apis/robot.md:6]: (opt) Also could update phrasing here

File: static/include/try-viam/create-a-reservation.md [hunk 3]
```diff
@@ -12,8 +13,8 @@
    When it is your turn to take over a Viam Rover, the page will show that it is configuring your rover.
    Once that is complete, you can click **TRY MY ROVER** to access your rented Viam Rover.
 
-   {{< alert title="Info" color="info" >}}
-   Viam creates a location called "Rover Rental" in your account's organization, and adds a Viam Rover with a basic config and a random name (like "wispy-shape") as a robot in this location.
+   {{< alert title="Info" color="info" >}}a machine
```
Comment (sguequierre) [inline static/include/try-viam/create-a-reservation.md:16]:
```suggestion
   {{< alert title="Info" color="info" >}}
```

**Review by sguequierre — CHANGES_REQUESTED** [review]
Couple small things otherwise LGTM

File: static/include/try-viam/create-a-reservation.md (hunk 3 above)
Comment (JessamyT) [reply static/include/try-viam/create-a-reservation.md:16]: Ooh weird bug--thanks v much for catching

File: static/include/services/apis/robot.md (hunk 2 above)
Comment (JessamyT) [reply static/include/services/apis/robot.md:6]:
```suggestion
| [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a machine that is connected to a machine over the given channel. |
```
Okay yeah--but while we're at it either way "robot connected to robot" or "machine connected to machine", this is actually kinda confusing and sounds circular--do you happen to know if there's a better way to phrase this overall? Not sure what it actually does

File: docs/build/program/apis/robot.md (hunk 1 above)
Comment (JessamyT) [reply docs/build/program/apis/robot.md:251]: Good point.
```suggestion
Create a RobotClient that is connected to a machine over the given channel.
```

File: docs/build/program/apis/robot.md [hunk 4]
```diff
@@ -210,14 +210,14 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 ### AtAddress
 
-Create a robot client that is connected to the robot at the provided address.
+Create a robot client that is connected to the machine at the provided address.
```
Comment (JessamyT) [inline docs/build/program/apis/robot.md:213]:
```suggestion
Create a RobotClient that is connected to the machine at the provided address.
```

File: static/include/services/apis/robot.md (hunk 2 above)
Comment (JessamyT) [reply static/include/services/apis/robot.md:6]:
```suggestion
| [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a RobotClient that is connected to a machine over the given channel. |
```
JK will use phrasing from the instance above. TY!

**Review by sguequierre — APPROVED** [review]

File: static/include/services/apis/robot.md [hunk 5]
```diff
@@ -2,17 +2,17 @@
 | Method Name                                                     | Description                                                                  |
 | --------------------------------------------------------------- | ---------------------------------------------------------------------------- |
 | [`Options.with_api_key`](/build/program/apis/robot/#optionswith_api_key) | Create Robot client connection options with an API key as credentials. |
```
Comment (JessamyT) [inline static/include/services/apis/robot.md:4]:
```suggestion
| [`Options.with_api_key`](/build/program/apis/robot/#optionswith_api_key) | Create RobotClient connection options with an API key as credentials. |
```

File: static/include/services/apis/robot.md [hunk 6]
```diff
@@ -2,17 +2,17 @@
...
-| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a robot client that is connected to the robot at the provided address. |
-| [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a robot that is connected to a robot over the given channel. |
-| [`Refresh`](/build/program/apis/robot/#refresh) | Manually refesh the underlying parts of the robot. |
-| [`Status`](/build/program/apis/robot/#status) | Get the status of the robot's components. |
-| [`GetOperations`](/build/program/apis/robot/#getoperations) | Get the list of operations currently running on the robot. |
-| [`CancelOperation`](/build/program/apis/robot/#canceloperation) | Cancel the specified operation on the robot. |
-| [`BlockForOperation`](/build/program/apis/robot/#blockforoperation) | Blocks on the specified operation on the robot. |
+| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a robot client that is connected to the machine at the provided address. |
```
Comment (JessamyT) [inline static/include/services/apis/robot.md:5]:
```suggestion
| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a RobotClient that is connected to the machine at the provided address. |
```
for consistency

**Pull request merged**